core/base/src/TROOT.cxx

```diff
@@ -2855,6 +2850,13 @@ void TROOT::IndentLevel()
    for (int i = 0; i < fgDirLevel; i++) std::cout.put(' ');
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Initialize ROOT explicitly.
+
+void TROOT::Initialize() {
```

Comment:
Nice.  Thanks.

Reply:
Actually, in a future PR, we should enforce that. We should not allow lazy initialization by typing `gROOT`. This usually leads to problems with teardown if the user is not careful where is the first use which is essentially explicit initialization and using `TROOT::Initialize` makes things much more readable.